apps/web-giddh/src/app/shared/header/header.component.ts

```diff
@@ -790,7 +788,7 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
                 if (isElectron) {
                     this.router.navigate(['/login']);
                 } else {
-                    window.location.href = (environment.production) ? `https://stage.giddh.com/login` : `https://test.giddh.com/login`;
+                    window.location.href = (environment.production) ? `https://giddh.com/login` : `https://test.giddh.com/login`;
```

Comment:
Need to check !!!!

Reply:
Whole file

Reply:
Please change : stage,test to giddh in whole file